src/scheduler/misc.c

```diff
@@ -242,17 +242,23 @@ add_str_to_array(char ***str_arr, char *str)
 
 /**
  * @brief
- * 		res_to_num - convert a resource string to a  sch_resource_t to
- *                      kilobytes
- *                      example: 1mb -> 1024
- *				 	1mw -> 1024 * SIZE_OF_WORD
+ * 		res_to_num - convert a resource string to a sch_resource_t to
+ *                      kilobytes or seconds or 0 for False & 1 for
+ *                      True
```

Comment:
So it might return 0 for false and 1 for true, 1 might just be 1 and 0 might just be 0.  Now that I think about this, why don't you just shorten this to 'convert a resource string to a numeric sch_resource_t'